website/static/js/profile.js

```diff
@@ -693,10 +731,19 @@ var JobViewModel = function() {
     DateMixin.call(self);
     TrackedMixin.call(self);
 
-    self.institution = ko.observable('').extend({required: true, trimmed: true});
     self.department = ko.observable('').extend({trimmed: true});
     self.title = ko.observable('').extend({trimmed: true});
 
+    self.institution = ko.observable('').extend({
+        trimmed: true,
+        required: {
+            onlyIf: function() {
+               return !!self.department() || !!self.title();
```

Comment:
The reason is that `!` is for negation, so your logic would be wrong. `!!` just makes truthy values evaluate to `true`.

```js
var truthyString = 'lol';
console.log(!!truthyString);  // true
```

In most cases, the `!!` doesn't make a difference.